testing/py-ycmd/APKBUILD

```diff
@@ -0,0 +1,390 @@
+# Contributor: Orson Teodoro <orsonteodoro@hotmail.com>
+# Maintainer: Orson Teodoro <orsonteodoro@hotmail.com>
+pkgname=py-ycmd
+_pkgname=ycmd
+pkgver=0_git20180224
+pkgrel=0
+pkgdesc="A code-completion & code-comprehension server"
+url="https://github.com/Valloric/ycmd"
+arch="aarch64 armhf ppc64le s390x x86 x86_64"
+install="$pkgname.post-install $pkgname.post-upgrade"
+_e_java= ; _e_c= ; _e_go= ; _e_js= ; _e_cs= ; _e_rust= ; _arch_makedepends= ;
+case "$CARCH" in
+	#java disabled but could be enabled if we have eclipse
+	aarch64*|ppc64le|arm*|s390x|x86|x86_64) _e_java= ; _e_c=1 ; _e_go=1 ; _e_js=1 ; _arch_makedepends="clang-dev clang-libs go nodejs nodejs-npm" ;;
+esac
+case "$CARCH" in
+	x86|x86_64) _e_cs=1 ; _arch_makedepends="$_arch_makedepends mono-dev" ;;
+esac
+case "$CARCH" in
+	x86_64)	_e_rust=1 ; _arch_makedepends="$_arch_makedepends rust cargo" ;;
+esac
+license="GPL-3.0-or-later Apache-2.0 MIT Python-2.0 GPL-2.1-or-later MPL-2.0 BSD-3-Clause BSD-2-Clause ZPL-2.1 ISC MS-PL EPL-1.0 CPL-1.0 W3C Apache-1.1 MPL-1.1 Plexus ICU custom"
+checkdepends="py-pyhamcrest py-webtest py-flake8 py-enum34 py-pycodestyle py-nose py-mock py-psutil py-nose-exclude py-requests py-urllib3 py-future py2-unittest2 !py2-bottle !py3-bottle"
```

Comment:
Why are you conflicting on packages in `$checkdepends`?

Reply:
The tests requires the internal bottle.  If you use the external py-bottle, it will trigger Shutdown_test FAILs.  The checkdepends stay up when there are some .makedepends-$pkgname and keep them up with the -K switch.  https://github.com/Valloric/ycmd/issues/670